runtime/rasdump/TextFileStream.cpp

```diff
@@ -51,10 +52,10 @@ void
 TextFileStream::open(const char* fileName, bool cacheWrites)
 {
 	PORT_ACCESS_FROM_PORT(_PortLibrary);
-	if (NULL != strstr(fileName, "/STDOUT/")) {
-		_FileHandle = 1; 
-	} else if (NULL != strstr(fileName, "/STDERR/")) {
-		_FileHandle = 2;
+	if (NULL != strstr(fileName, J9PRINT_TO_STDOUT)) {
+		_FileHandle = J9PORT_TTY_OUT; 
+	} else if (NULL != strstr(fileName, J9PRINT_TO_STDERR)) {
+		_FileHandle = J9PORT_TTY_ERR;
```

Comment:
This should use the same names as below (`OMRPORT_TTY_OUT ` and `OMRPORT_TTY_ERR `).